fairroot/tools/FairBaseTypes.h

```diff
@@ -0,0 +1,81 @@
+/********************************************************************************
+ * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairBaseTypes.h
+ *
+ *  Created on: May 16, 2023
+ *      Author: R.Karabowicz
+ */
+
+#ifndef FAIR_BASETYPES_H_
+#define FAIR_BASETYPES_H_
+
+#include <iostream>
+#include <limits>
+
+namespace FairRoot {
+    struct EntryID {
```

Comment:
Consider to add some Doxygen comments to the namespace and class?

```suggestion
/**
 * \brief Main namespace for FairRoot classes / types
 *
 * All new public classes and types of FairRoot will be in this namespace.
 * Note: Nost classes are still in no namespace.
 */
namespace FairRoot {

/**
 * \brief <What Entry is this ID meant for?>
 */
struct EntryID
{
```